src/Wallabag/CoreBundle/Resources/views/themes/material/Entry/entries.html.twig

```diff
@@ -38,6 +51,21 @@
     </div>
 
     <ul class="{% if listMode == 1 %}collection{% else %}row data{% endif %}">
+
+        <li class="mass-buttons">
+            {% if entries.count > 0 and listMode == 1 %}
+                <span>
+                    <input id="selectAll" type="checkbox" onClick="toggle(this)" />&nbsp;
```

Comment:
Can't you use something less ugly than the `onClick`? Maybe in `app/Resources/static/themes/material/index.js`?

Reply:
I did a fixup if you want to check it.